### ISSUE TITLE
[next/image] flash broken image placeholder (alt text and white border) before load image

### ISSUE THREAD

**Issue by shashkovdanil** [vercel/next.js#18448]
# Bug report

## Describe the bug

I use next/image component and after reload page I see blink alt (attribute `alt`) text.

## To Reproduce

1. Use next/image component
2. Reload page

## Expected behavior

1. Show alt text if image didn't load (404 error for example)
2. If I have good image I shouldn't see flash broken image placeholder

## Screenshots

https://monosnap.com/file/BjNLnRZMWmwG5NugYk8TuDm3VZ6tez

**screen from video above (timecode 8sec)**
![Screenshot 2020-10-29 133243](https://user-images.githubusercontent.com/19692986/97544372-50e31380-19eb-11eb-8330-2a924daad1c0.png)

## System information

- OS: Windows 10
- Browser: Latest Edge, Latest Firefox
- Version of Next.js: 10
- Version of Node.js: 14.13

## Additional context

My code:

```jsx
import Link from 'next/link'
import Image from 'next/image'
import slugify from '@sindresorhus/slugify'
import format from 'date-fns/format'
import isToday from 'date-fns/isToday'
import isTomorrow from 'date-fns/isTomorrow'
import ru from 'date-fns/locale/ru'
import cx from 'classnames'
import { ReleaseType, ReleaseInList } from 'types/common'
import PlatformList from '../PlatformList'
import Text from '../Text'
import ExpectButton from '../ExpectButton'

import styles from './styles.module.css'

export enum Source {
  Calendar = 'calendar',
  Today = 'today',
  Profile = 'profile',
}

function renderDate(today: boolean, tomorrow: boolean, date: Date) {
  if (today) return 'сегодня'

  if (tomorrow) return 'завтра'

  return format(date, 'EEEEEE, d MMM', {
    locale: ru,
  })
}

interface Props {
  release: ReleaseInList
  source: Source
}

function ReleaseCard({ release, source }: Props) {
  const { title, released, release_id } = release

  const releasedDate = new Date(released)

  const slug = slugify(title)
  const today = isToday(releasedDate)
  const tomorrow = isTomorrow(releasedDate)

  return (
    <Link href="/release/[id]" as={`/release/${release_id}-${slug}`}>
      <a className={styles.ReleaseCard}>
        <div className={styles.Header}>
          <div
            className={cx(styles.Date, {
              [styles.isToday]: today,
            })}
          >
            {renderDate(today, tomorrow, releasedDate)}
          </div>
          <ExpectButton release={release} />
        </div>
        <div className={styles.Cover}>
          <Image src={release.cover} alt={release.title} unsized />
        </div>
        <div className={styles.Footer}>
          <Text className={styles.Title}>{release.title}</Text>
          {release.type === ReleaseType.Films && (
            <Text secondary i className={styles.Info}>
              {release.director}
            </Text>
          )}
          {release.type === ReleaseType.Series && (
            <Text secondary i className={styles.Info}>
              {release.season} сезон
            </Text>
          )}
          {release.type === ReleaseType.Games && (
            <PlatformList platforms={release.platforms} />
          )}
        </div>
      </a>
    </Link>
  )
}

export default ReleaseCard
```

```css
.Cover {
  display: flex;
  width: 100%;
  height: 100%;
  background-color: #000;
  transition: opacity 250ms;

  &::after {
    position: absolute;
    top: 20%;
    left: 0;
    width: 100%;
    height: 80%;
    background-image: var(--release-card-cover-gradient);
    content: '';
  }

  * {
    width: 100%;
    height: 100%;
  }

  img {
    object-fit: cover;
  }
}
```

When I use [lazysizes](https://github.com/aFarkas/lazysizes) I had similar problem, but I fix it like this

You can check [it](https://github.com/aFarkas/lazysizes#broken-image-symbol)

```jsx
const IMG_PLACEHOLDER = 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=='

function Image({
  lazy = true,
  src,
  alt = '',
  ...rest
}: Props & HTMLAttributes<HTMLImageElement>) {
  const isAmp = useAmp()

  if (isAmp) {
    return <amp-img src={src} alt={alt} layout="fill" {...rest} />
  }

  if (lazy) {
    return (
      <img
        className="lazyload"
        src={IMG_PLACEHOLDER} // It is fix that problem
        data-src={src}
        alt={alt}
        {...rest}
      />
    )
  }

  return <img  src={src} alt={alt} {...rest} />
}
```

**Comment by Timer** [comment]
Please provide a full reproduction so we can take a look!

**Comment by shashkovdanil** [comment]
@Timer, it reproduce if I add alt text to images, but it important part for SEO

**Repo:** https://github.com/shashkovdanil/reproduce-nextjs-bug


**Flash moment:**
![Screenshot 2020-10-31 130839](https://user-images.githubusercontent.com/19692986/97774508-45722280-1b7a-11eb-9270-954852210489.png)

**After ~1-2ms**
![Screenshot 2020-10-31 130856](https://user-images.githubusercontent.com/19692986/97774509-473be600-1b7a-11eb-971e-3bef8f9ab85a.png)

**Comment by styfle** [comment]
I believe this is fixed on canary.

Please visit https://image-component.nextjs.gallery or try [this example](https://github.com/vercel/next.js/tree/canary/examples/image-component) locally to confirm.

**Comment by styfle** [comment]
This was fixed in PR #18903

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
